src/cli.rs

```diff
@@ -153,6 +157,7 @@ pub struct Cli {
     ///
     /// The environment variables `http_proxy` and `https_proxy` can also be used, but
     /// are completely ignored if --proxy is passed.
+    /// {n}{n}{n}
```

Comment:
By the way, this is not true at the moment, right?
```
/// The environment variables `http_proxy` and `https_proxy` can also be used, but
/// are completely ignored if --proxy is passed.
```

Reply:
It is true, but that's part of reqwest's behavior. Calling `.proxy()` stops it from looking at the environment variables later on: https://github.com/seanmonstar/reqwest/blob/a8566383168c0ef06c21f38cbc9213af6ff6db31/src/async_impl/client.rs#L582